packages/bot-skeleton/src/services/api/ticks_service.js

```diff
@@ -46,6 +46,9 @@ export default class TicksService {
         this.tickListeners = new Map();
         this.ohlcListeners = new Map();
         this.subscriptions = new Map();
+        this.tick_history_promise = null;
+        this.active_symbol_promise = null;
```

Comment:
Could we keep the names of the calls (plural)? e.g.

```JavaScript
        this.ticks_history_promise = null;
        this.active_symbols_promise = null;
```

Reply:
done